pkg/api/v1/resource/helpers.go

```diff
@@ -65,37 +68,75 @@ func PodRequestsAndLimits(pod *v1.Pod) (reqs, limits v1.ResourceList) {
 		maxResourceList(reqs, container.Resources.Requests)
 		maxResourceList(limits, container.Resources.Limits)
 	}
+
+	// if PodOverhead feature is supported, add overhead for running a pod
+	// to the non-zero sum of container requests/limits:
```

Comment:
I had gone back and forth on this.  I think it makes sense not to add it, since it would effectively be effectively changing the QoS, right?

While the original spec would have this field as best-effort, we'd be providing requests, slightly changing the eviction behavior between a pod utilizing pod overhead and not.

I've gone back and forth, so am happy to hear your input on this.

Reply:
addressed